tests/foreman/maintain/test_backup_restore.py

```diff
@@ -452,13 +452,14 @@ def test_positive_backup_restore(
 
     expected_files = get_exp_files(sat_maintain, skip_pulp)
     assert set(files).issuperset(expected_files), assert_msg
-
+    result = sat_maintain.execute(
+        f"tar -tvf {backup_dir}/config_files.tar.gz | grep -E '{sat_maintain.hostname}|ansible.cfg'"
```

Comment:
if you drop the `-v` to `tar`, you get only the filenames, so you can do something like this:
```ruby
config_files = sat_maintain.execute(f"tar -tvf {backup_dir}/config_files.tar.gz").stdout.splitlines()  # this gives you one filename per list entry
assert 'etc/ansible/ansible.cfg' in config_files  # this asserts an entry in the list, instead of a huge string
assert f'root/{sat_maintain.hostname}-certs.tar' in config_files  # but it now requires full-paths to match, so this one will be slightly more verbose
```